libraries/grpc-sdk/src/modules/config/index.ts

```diff
@@ -103,14 +108,14 @@ export class Config extends ConduitModule<typeof ConfigDefinition> {
     };
     const self = this;
     this.client!.moduleHealthProbe(request)
```

Comment:
This should typically be replaced by the health checks. Maybe the entire complexity of this (including the service/clientNames) could be solved by merging the core and config packages. Since the Core only bootstraps stuff but it's very dependent on config. So merging Config and Core would be a reasonable move. This also brings us closer to also enabling the optional splitting (during deployment) of core/config and the routers, but I'm getting off-topic here.